sotodlib/tod_ops/detrend.py

```diff
@@ -1,10 +1,19 @@
 import numpy as np
 
-def detrend_data(tod, method='linear', axis_name='samps', 
-                 signal_name='signal', count=10):
+def detrend_tod(
+    tod, 
+    method='linear', 
+    axis_name='samps', 
+    signal_name='signal', 
+    in_place=True,
+    wrap_name=None,
+    count=10
+):
     
-    """Returns detrended data. Decide for yourself if it goes
-        into the axis manager. Generally intended for use before filter.
+    """Returns detrended data. Detrends data in place by default but pass
+        in_place=False if you would like a copied array (such as if you're just
```

Comment:
Un-indent second (and all subsequent) lines of docstring.

In argument descriptions, please capitalize start of sentences after periods.

The docstring style here isn't quite numpy nor google... but sphinx seems ok with it, and it matches the style of other tod_ops docs.  So I won't fight that.